### ISSUE TITLE
add chapter on deprecating a data package?

### ISSUE THREAD

**Issue by cgries** [EDIorg/data-package-best-practices#74]
use case: long-term observation dataset was archived as two packages, in the latest update the author wants most tables in one package and get rid of others. And updated tables have slightly different structures.

**Comment by rfbrown** [comment]
Good idea. Another use case I've been thinking about at MCM: we currently archive monthly and/or daily summaries of our met and hydro data as separate packages from the high-frequency data. So this means for each met station or stream gage, we have 2-3 packages per station. It's a ton of extra work on my part, and creates confusion to our researchers (they often download the wrong package). I would like to consider consolidating into a single package per station, but I also wonder how others are tackling? If we consolidate, then we'll have to depreciate some packages...
